### ISSUE TITLE
[IMPROVE] Add timezone info in `candle --raw`

### ISSUE THREAD

**Issue by hjoaquim** [OpenBB-finance/OpenBBTerminal#4175]
^
Discussed here: #4158

**Comment by hjoaquim** [comment]
Tagging the intervenients: @jmaslek @deeleeramone

**Comment by deeleeramone** [comment]
Agreed. This is a requirement for simply looking at single-day intraday data. The timestamp, however, should be pinned to the timezone that the exchange operates in.
